flutter-for-android.md

```diff
@@ -0,0 +1,1845 @@
+         child: new GestureDetector(
+           child: new FlutterLogo(
+             size: 200.0,
+           ),
+           onTap: () {
+             print("tap");
+           },
+         ),
+       ));
+     }
+   }
+   {% endprettify %}
+
+**How do I handle other gestures on widgets?**
+
+Using the GestureDetector we can listen to a wide range of Gestures such as
+
+- Tap
+
+  - `onTapDown` A pointer that might cause a tap has contacted the screen at a 
+    particular location.
+  - `onTapUp` A pointer that will trigger a tap has stopped contacting the 
+    screen at a particular location.
+  - `onTap` A tap has occurred.
+  - `onTapCancel` The pointer that previously triggered the `onTapDown` will 
+    not end up causing a tap.
+
+- Double tap
+
+  - `onDoubleTap` The user has tapped the screen at the same location twice in 
+    quick succession.
+
+- Long press
+
+  - `onLongPress` A pointer has remained in contact with the screen at the same 
+    location for a long period of time.
+
+- Vertical drag
+
+  - `onVerticalDragStart` A pointer has contacted the screen and might begin to 
+    move vertically.
+  - `onVerticalDragUpdate` A pointer that is in contact with the screen and 
+    moving vertically has moved in the vertical direction.
+  - `onVerticalDragEnd` A pointer that was previously in contact with the 
+    screen and moving vertically is no longer in contact with the screen and was 
+    moving at a specific velocity when it stopped contacting the screen.
+
+- Horizontal drag
+
+  - `onHorizontalDragStart` A pointer has contacted the screen and might begin 
+    to move horizontally.
+  - `onHorizontalDragUpdate` A pointer that is in contact with the screen and 
+    moving horizontally has moved in the horizontal direction.
+  - `onHorizontalDragEnd` A pointer that was previously in contact with the 
+    screen and moving horizontally is no longer in contact with the screen and was 
+    moving at a specific velocity when it stopped contacting the screen.
+
+
+
+For example here is a GestureDetector for double tap on the FlutterLogo that 
+will make it rotate
+
+<!-- skip -->
+{% prettify dart %}
+AnimationController controller;
+CurvedAnimation curve;
+
+@override
+void initState() {
+  controller = new AnimationController(duration: const Duration(milliseconds: 2000), vsync: this);
+  curve = new CurvedAnimation(parent: controller, curve: Curves.easeIn);
+}
+
+@override
+Widget build(BuildContext context) {
+  return new Scaffold(
+      body: new Center(
+    child: new GestureDetector(
+      child: new RotationTransition(
+          turns: curve,
+          child: new FlutterLogo(
+            size: 200.0,
+          )),
+      onDoubleTap: () {
+        if(controller.isCompleted){
+          controller.reverse();
+        }else{
+          controller.forward();
+        }
+    
+      },
+    ),
+  ));
+}
+{% endprettify %}
+
+# Listviews & Adapters
+
+**What is the alternative to a ListView in Flutter?**
```

Comment:
You have other answers above that mention listviews. Should we put them together?